libs/media-messages/schemas/payloads/media/media-service-asyncapi.yml

```diff
@@ -86,38 +86,56 @@ channels:
 components:
   messages:
     check-finish-ingest-item-command:
+      tags:
+        - name: aggregate-type:ingest-item
       contentType: application/json
       payload:
         $ref: 'commands/check-finish-ingest-item-command.json'
     check-finish-ingest-document-command:
+      tags:
+        - name: aggregate-type:ingest
       contentType: application/json
       payload:
         $ref: 'commands/check-finish-ingest-document-command.json'
     delete-entity-command:
+      tags:
+        - name: aggregate-type:entity
       contentType: application/json
       payload:
         $ref: 'commands/delete-entity-command.json'
     publish-entity-command:
+      tags:
+        - name: aggregate-type:entity
       contentType: application/json
       payload:
         $ref: 'commands/publish-entity-command.json'
     start-ingest-command:
+      tags:
+        - name: aggregate-type:ingest
       contentType: application/json
       payload:
         $ref: 'commands/start-ingest-command.json'
     start-ingest-item-command:
+      tags:
+        - name: aggregate-type:ingest-item
       contentType: application/json
       payload:
         $ref: 'commands/start-ingest-item-command.json'
     unpublish-entity-command:
+      tags:
+        - name: aggregate-type:entity
       contentType: application/json
       payload:
         $ref: 'commands/unpublish-entity-command.json'
     update-metadata-command:
+      tags:
+        - name: aggregate-type:ingest-item
```

Comment:
Very minor, but can also be `ingest-item-step` (we do process each step separately)

Reply:
Actually, to align with
`services\media\service\src\ingest\handlers\start-ingest-item-handler.ts` (using
entity_id as an aggregate-id) - this should be `aggregate-type:entity`.

Reply:
Good point - adjusted the code to use the item ID instead to rather be in line with the other ingests.